src/modules/analysis/AnalysisAddComponent.jsx

```diff
@@ -55,29 +88,42 @@ function newLine() {
   </Form>;
 }
 
+const getVal = (form, field) => form[field] && (form[field].rawValue || '');
```

Comment:
this method is unnecessary. There is always a form. But there might not be a form[field] .... you **can** remove this method and just inline ```form.doneDate.value || ''``` etc in the component below